sirepo/package_data/jupyterhub_templates/page.html

```diff
@@ -0,0 +1,20 @@
+{% extends "templates/page.html" %}
+{% block nav_bar_right_items %}
+<li>
+    <a id="sirepo-slack-link" href="https://slack.sirepo.com">
+         <span>
+             <img style="filter: invert(0.5);" width="70" src="/static/svg/slack.svg" title="Join us on Slack"/>
+        </span>
+    </a>
+</li>
+<script>
+const SIREPO = {};
+</script>
+<script src="/auth-state">
```

Comment:
Put a jinja comment `{# #}` with a POSIT for where this is defined. Unfortunately, you can't put comments in JSON so you can't POSIT in schema-common.json.